adapters/seedingAlliance/seedingAlliance.go

```diff
@@ -127,19 +142,25 @@ func curExists(allowedCurrencies []string, newCurrency string) bool {
 	return false
 }
 
-func addTagID(imp *openrtb2.Imp) error {
+func getExtInfo(imp *openrtb2.Imp) (string, error) {
 	var ext adapters.ExtImpBidder
 	var extSA openrtb_ext.ImpExtSeedingAlliance
 
+	seatID := "pbs"
+
 	if err := json.Unmarshal(imp.Ext, &ext); err != nil {
-		return fmt.Errorf("could not unmarshal adapters.ExtImpBidder: %w", err)
+		return "", fmt.Errorf("could not unmarshal adapters.ExtImpBidder: %w", err)
 	}
 
 	if err := json.Unmarshal(ext.Bidder, &extSA); err != nil {
-		return fmt.Errorf("could not unmarshal openrtb_ext.ImpExtSeedingAlliance: %w", err)
+		return "", fmt.Errorf("could not unmarshal openrtb_ext.ImpExtSeedingAlliance: %w", err)
 	}
 
 	imp.TagID = extSA.AdUnitID
 
-	return nil
+	if extSA.SeatID != "" {
+		seatID = extSA.SeatID
+	}
```

Comment:
Could you please add a json test case where this if condition is tested? You might need to have an imp that has SeatId set that will replace your default `pbs` value. 

Json test cases are recommended to test a bid adapter. [Refer to this section of documentation](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#adapter-code-tests) for more information.

Reply:
Added a json test case for a native request with seat ID